nexuse2e-core/src/main/java/org/nexuse2e/util/CertificateUtil.java

```diff
@@ -490,6 +490,19 @@ public static String getRemainingValidity(X509Certificate x509certificate) {
         return remaining;
     } // getRemainingValidity
 
+    /**
+     * @param x509certificate
+     * @return int RemainingDayCount
+     */
+    public static int getRemainingDayCount(X509Certificate x509certificate) {
+        long nowMils = System.currentTimeMillis();
+        long expMils = x509certificate.getNotAfter().getTime();
+
+        long div = expMils - nowMils;
```

Comment:
Can we give div a better name? Something like validMils

Reply:
Done.